<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FolderContentsDialog</class>
 <widget class="QDialog" name="FolderContentsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Folder contents</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="labelTotal">
     <property name="text">
      <string>Total</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="checkBox_Top10">
     <property name="text">
      <string>Top10</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="labelFolderName">
     <property name="text">
      <string>folder name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QTreeWidget" name="treeWidget">
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>Extension</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Files Number</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Total Size</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
